index.bs

```diff
@@ -3921,20 +3922,19 @@ follows:</p>
 
 <h2 id=rendering>Rendering</h2>
 
-<p class="note">This section describes in some detail how to visually render WebVTT cues in a user
-agent. The processing model is quite tightly linked to media elements in HTML. When supporting
-WebVTT in media players that don't support CSS, equivalent visual rendering will need to be
-implemented.</p>
+<p>This section describes how to visually render WebVTT cues in a user agent. The processing model
+is tightly linked to media elements in HTML and follows the handling of cues when <a>playing the
+media resource</a>. When supporting WebVTT in media players that don't support CSS, equivalent
```

Comment:
"don't support CSS" now needs to be expanded to "don't support CSS and HTML5" I think

Reply:
HTML5 is being used to define semantics of cue times; do I need to "support HTML5" to get that?

Reply:
@nigelmegitt how about "don't support HTML nor CSS" - I just want to make sure players that support CSS but not HTML are covered.

Reply:
Sorry @dwsinger I didn't spot your response a week ago. I think that you do not need to support HTML5 to match the semantics of cue times, in theory, though you might in practice!

Reply:
@silviapfeiffer the correct wording would be "that support neither HTML nor CSS" I think, or alternatively "that do not support HTML and do not support CSS" to be super clear.

Reply:
@silviapfeiffer sorry for the confusion, but I think we would need to clarify what implementations that support HTML but not CSS, or CSS but not HTML would do with my proposed wording at https://github.com/w3c/webvtt/pull/419#discussion_r170596448 and also in the more recent update. I suspect the better option is to say "that do not support both HTML and CSS".